MediaBrowser.Model/Entities/MetadataProvider.cs

```diff
@@ -17,11 +17,6 @@ public enum MetadataProvider
         /// </summary>
         Tmdb = 3,
 
-        /// <summary>
-        /// The TVDB.
-        /// </summary>
-        Tvdb = 4,
```

Comment:
I don't agree with this removal.

Reply:
Yes, this enum is used in plugins.

Reply:
This is superseded by https://github.com/jellyfin/jellyfin/pull/4298